packages/fiori-freestyle-writer/test/__snapshots__/basic.test.ts.snap

```diff
@@ -1024,8 +1024,8 @@ builder:
     "state": "modified",
   },
   "webapp/Component.ts": Object {
-    "contents": "import BaseComponent from \\"sap/ui/core/UIComponent\\";
-import { createDeviceModel } from \\"nods1/model/models\\";
+    "contents": "import BaseComponent from 'sap/ui/core/UIComponent';
```

Comment:
Not sure the double to single quotes should be applied here. At least for JS projects double quotes should be used. TS should be the same.